src/main/resources/application.properties

```
@@ -0,0 +1,26 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/comexdb
+spring.datasource.username=larilabanca
+spring.datasource.password=12345678
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+# jpa
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
```

Você está com o MySQL 8? Aí, o legal seria usar essa configuração aqui:

```suggestion
spring.jpa.database-platform=org.hibernate.dialect.MySQL8Dialect
```